tools/toolchain/scripts/install_libxc.sh

```diff
@@ -30,12 +30,13 @@ case "$with_libxc" in
                 echo "libxc-${libxc_ver}.tar.gz is found"
             else
                 download_pkg ${DOWNLOADER_FLAGS} ${libxc_sha256} \
-                             https://www.cp2k.org/static/downloads/libxc-${libxc_ver}.tar.gz
+                             https://gitlab.com/libxc/libxc/-/archive/${libxc_ver}/libxc-${libxc_ver}.tar.gz
             fi
             echo "Installing from scratch into ${pkg_install_dir}"
             [ -d libxc-${libxc_ver} ] && rm -rf libxc-${libxc_ver}
             tar -xzf libxc-${libxc_ver}.tar.gz
             cd libxc-${libxc_ver}
+            autoreconf -i > autoreconf.log 2>&1
```

Comment:
The tarball at https://www.tddft.org/programs/libxc/download/ seems to contain `configure` script, could we use that one instead? This requires the user to have a full autoconf/automake/m4 toolchain installed.

Reply:
... but that's what you probably meant by "adapted to the hotfix" I guess?

Reply:
No. First, I was using the github tarball. The other one seemed to be published this morning. It works locally, now with the other tarball.